scripts/electronzip.js

```diff
@@ -0,0 +1,46 @@
+const fsSync = require('fs');
+const fs = fsSync.promises;
+const path = require('path');
+const archiver = require('archiver');
+
+const baseDir = path.join(__dirname, '..');
+const buildDir = path.join(baseDir, 'build');
+const distDir = path.join(baseDir, 'dist');
+
+(async () => {
+  let distFolders = [];
+  try {
+    distFolders = await fs.readdir(distDir);
+  } catch (e) {
+    await fs.mkdir(distDir);
+  }
+  for (const oldZip of distFolders) {
+    await fs.unlink(path.join(distDir, oldZip));
+  }
+
+  const folders = await fs.readdir(buildDir);
```

Comment:
This script would throw if being run before `electronpackage` as `build` wouldn't exist yet, right?
```suggestion
  try {
    const folders = await fs.readdir(buildDir);
  } catch (e) {
    console.error("Run "npm run electronpackage" before zipping');
    return;
  }
```

Reply:
Right, this script was only intended to be run on CI so I left this check out.